validation/src/main/scala/hmda/validation/SubmissionLarStats.scala

```diff
@@ -40,15 +43,26 @@ class SubmissionLarStats(submissionId: SubmissionId) extends HmdaPersistentActor
   }
 
   override def receiveCommand: Receive = super.receiveCommand orElse {
+    case s: String =>
+      totalSubmittedLars = totalSubmittedLars + 1
+
     case LarValidated(_, _) =>
-      totalLars = totalLars + 1
+      totalValidatedLars = totalValidatedLars + 1
+
+    case CountSubmittedLarsInSubmission =>
```

Comment:
I could use help understanding why we need two different counts--one for "submitted" lars, and one for "validated" lars. Under what circumstances would those counts be different for a single submission?